packages/java/neo-auth-service/README.md

```diff
@@ -26,12 +26,39 @@ The `TokenService` class provides methods for generating and verifying JWTs. To
 
 ## Running Tests
 
-Actually, we don't have any test.
+You need to have gradle on your project.
+You need to open the terminal and run this command 
```

Comment:
Mejor:

To run the tests, open a terminal in the root of the project and execute the following command: